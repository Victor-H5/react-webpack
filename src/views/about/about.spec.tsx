import React from 'react';
import { shallow } from 'enzyme';
import About from './about';

describe('About component', () => {
	it('should match snapshot', () => {
		const wrapper = shallow(<About />);
		expect(wrapper).toMatchSnapshot();
	});

	it('should has click', () => {
		const wrapper = shallow(<About />);
		expect(wrapper.contains('click')).toBe(true);
	});
});
