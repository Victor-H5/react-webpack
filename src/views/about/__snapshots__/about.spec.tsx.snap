// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component should match snapshot 1`] = `
<div>
  <button
    onClick={[Function]}
  >
    click
  </button>
  <Hello
    compiler="TypeScript"
    framework="React"
  />
</div>
`;
